docs/src/pages/components/slider/slider.md

```diff
@@ -48,6 +48,22 @@ Continuous sliders allow users to select a value along a subjective range.
 
 {{"demo": "pages/components/slider/VerticalSlider.js"}}
 
+## Track
+
+The track shows the range available for user selection.
+
+### Removed track
+
+The track can be turned off with `track={false}`.
+
+{{"demo": "pages/components/slider/TrackFalseSlider.js"}}
+
+### Inverted track
+
+The track can be inverted with `track="inverted"`.
```

Comment:
A use-case description would be helpful.

Reply:
Agree

Reply:
I would have provided one, but couldn't think of one that aligned with the example.

Reply:
Well, I think that we can merge even if we don't find a better description. I have nothing that comes to mind.